Assets/VRTK/Scripts/VRTK_PlayerJump.cs

```diff
@@ -0,0 +1,366 @@
+// Player Jump|Scripts|0240
+namespace VRTK
+{
+    using UnityEngine;
+    using System.IO;
+    using System;
+
+    /// <summary>
+    /// The player jump provides a jump behavior by tracking HMD world position over time, calculating velocities of a crouch and a spring motion, and adding velocity to the attached rigidbody on playerpresence.  It should be attached to the `[CameraRig]` object.
+    /// </summary>
+    /// <remarks>
+    /// This relies heavily on the status of the playerPresence, and will probably have to be updated as that changes.  In particular, playerPresence sometimes deactivates physics, which can leave this in a stuck state
```

Comment:
I wouldn't mention that things may change. Write the docs as if this is the way it's always going to work. If things change, we'll update the docs to reflect the changes